vars/chart.groovy

```diff
@@ -0,0 +1,53 @@
+// public
```

Comment:
I think the name of this file would be better as `helm` as opposed to `chart` just so it's clear as to what type of chart.  Then we could have functions such as `helm.publishChart`, helm.deployChart` etc

Reply:
Agreed. I've made the change for `helm` and a couple of others, as suggested.